service/history/timerQueueStandbyTaskExecutor.go

```diff
@@ -139,11 +139,10 @@ func (t *timerQueueStandbyTaskExecutor) executeUserTimerTimeoutTask(
 			); isExpired {
 				return getHistoryResendInfo(mutableState)
 			}
-			// since the user timer are already sorted, so if there is one timer which will not expired
-			// all user timer after this timer will not expired
-			break Loop //nolint:staticcheck
+			// Since the user timers are already sorted, so if there is one timer which is not expired
+			// all user timers after this timer are not expired.
```

Comment:
```suggestion
			// Since the user timers are already sorted, then if there is one timer which is not expired,
			// all user timers after that timer are not expired.
```